src/main/java/frc/robot/Robot.java

```diff
@@ -103,81 +79,10 @@ public void disabledInit() {
         Shuffleboard.stopRecording();
     }
 
-    public Command goodFlip() {
-        CommandChain retValue = new CommandChain("Good Flip");
-
-        retValue.then(lift.goToAtLeast(Heights.kLiftFlipHeight)).then(maflipulator.crappyFlip());
-        return retValue;
-    }
-
-    public Command stowAndGo() {
-        CommandChain retValue = new CommandChain("Stow it and go!");
-        retValue.then(lift.goToHeight(Heights.kFloorLoad), maflipulator.stowAndGoPosition());
-        return retValue;
-    }
-
-    public Command goToLoadingStation() {
-        CommandChain retValue = new CommandChain("Go to Loading Station");
-
-        retValue.then(lift.goToHeight(Heights.kLoadingStation));
-        return retValue;
-
-    }
-
-    public Command goToRocketMiddleHatch() {
-        CommandChain retValue = new CommandChain("Go to Rocket Middle Hatch");
-
-        retValue.then(lift.goToHeight(Heights.kRocketHatchMid), maflipulator.goToScoringPosition());
+    public CommandChain rocketAuto() {
```

Comment:
This will very much not work as it exists, can we remove it?

Reply:
done